src/calculateRentalCost.js

```diff
@@ -21,14 +21,20 @@
  *  - calculateRentalCost(1) === 40
  *  - calculateRentalCost(3) === 100
  *  - calculateRentalCost(7) === 230
- *
- * @param {number} days
- *
- * @return {number}
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
```

Comment:
Create constant variables for such magic numbers as 40 and 50 with a meaningful name. Such approach improves the readability and maintenance of your code